src/state/ducks/moduleDuck.js

```diff
@@ -88,6 +89,27 @@ export const moduleActions = {
       .catch(err => thunkFail(err.message))
       .finally(() => thunkResolve());
   },
+  // Create module
+  createModuleThunk: newModule => dispatch => {
+    const {
+      thunkStart,
+      thunkFail,
+      thunkResolve,
+    } = moduleThunkUtils.getTriggersFromPrefix(dispatch, 'create');
+
+    thunkStart();
+
+    axiosAuth()
+      .post('/modules', newModule)
+      .then(res => {
+        dispatch({ status: CREATE_MODULE_SUCCESS, payload: res.data });
+        console.log(res.data);
```

Comment:
remove `console.log`

Reply:
fixed, I'm so sorry about all these. Thank you for catching them